src/pages/demo/expandableText/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { drupalClient } from '@/utils/drupalClient'
+import { GetStaticPropsContext, GetStaticPropsResult } from 'next'
+import { DrupalNode } from 'next-drupal'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import Container from '@/components/container'
+import ExpandableText from '@/components/paragraph/expandable_text'
+
+interface ExpandableTextPageProps {
+  expandableTextCollection: DrupalNode[]
```

Comment:
This should be `DrupalParagraph[]`...underlying base data structure is a little different.